lib/ffmpeg/transcoderbase.py

```diff
@@ -128,9 +128,13 @@ def gen_input_opts(self):
       opts += f" -hwaccel {hw}"
     else:
       opts = "-init_hw_device {hwaccel}={hwdevice}"
+      if vars(self).get("dhwframes", None) is not None:
```

Comment:
How about use var name `ihwframes`, instead, since the function name refers to "input" opts?  It would be more consistent with how we named other vars (e.g. ihwformat vs. ohwformat) in places like vpp tests.